dind/README.md

```diff
@@ -0,0 +1,46 @@
+# rke DinD
+
+Docker-in-Docker (DinD) feature is created mainly for testing.
+
+DinD creates docker containers locally which are then used as cluster nodes.
+
+It still requires a cluster.yml with minimal information:
+
+```
+kubernetes_version: "$KUBE_VERSION"
+nodes:
+- address: foo
+  role:
+  - controlplane
+  - worker
+  - etcd
+  user: bar
+```
+
+To create a cluster with DinD execute:
+`docker up --dind`
+
+After the creation is completed, you will see one docker container per node running. You can use the created `kube_config_cluster.yml` to access the cluster as in a normal rke deployment
+
+To remove a cluster with DinD execute:
+`docker remove --dind`
+
+The docker containers should be removed completely
+
+# FAQ
+
+* Do I need to create a network for the docker containers?
+
+No. The created docker containers use the default docker bridge network which allows to connect from the host to the containers on IPs in the subnet "172.17.0.0/16". However, they have no ssh service running, therefore you `docker exec` if you want to connect to them
+
+* What image is used for those containers?
+
+There is a special image: rancher/docker:19.03.12-dind that is used for that
```

Comment:
Where does this come from? We are at 20.10.23 (https://github.com/rancher/rke/blob/release/v1.5/dind/dind.go#L16) and it's not a special image, it is from upstream (https://hub.docker.com/_/docker)

Reply:
When using rke from release 1.4.10, I see in the logs:
```
INFO[0000] Running RKE version: v1.4.10                 
INFO[0000] Pulling image [docker:19.03.12-dind] on host [unix:///var/run/docker.sock], try #1 
INFO[0017] Image [docker:19.03.12-dind] exists on host [unix:///var/run/docker.sock] 
```

Reply:
This PR is on the `release/v1.5` branch which uses a different version. I would remove the version and link users to the file where the image is specified so they can look it up themselves.

It's still not a special image, its the upstream Docker image.